src/views/TechnicalAssistance/RequestForm/Feedback.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Grid, GridContainer, IconArrowBack } from '@trussworks/react-uswds';
+import { sortBy } from 'lodash';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import PageHeading from 'components/PageHeading';
+import Divider from 'components/shared/Divider';
+import { GetTrbRequest_trbRequest as TrbRequest } from 'queries/types/GetTrbRequest';
+import { TRBFeedbackAction } from 'types/graphql-global-types';
+import { formatDateLocal } from 'utils/date';
+
+import Breadcrumbs from '../Breadcrumbs';
+
+type FeedbackProps = {
+  request: TrbRequest;
+  taskListUrl: string;
+};
+
+function Feedback({ request, taskListUrl }: FeedbackProps) {
+  const { t } = useTranslation('technicalAssistance');
+
+  const returnUrl = `/trb/requests/${request.id}/check`;
+
+  const returnToFormLink = useMemo(
+    () => (
+      <UswdsReactLink to={returnUrl}>
+        <IconArrowBack className="margin-right-1 text-middle" />
+        <span className="line-height-body-5">
+          {t('requestFeedback.returnToForm')}
+        </span>
+      </UswdsReactLink>
+    ),
+    [returnUrl, t]
+  );
+
+  return (
+    <GridContainer className="width-full">
+      <Breadcrumbs
+        items={[
+          { text: t('heading'), url: '/trb' },
+          {
+            text: t('taskList.heading'),
+            url: taskListUrl
+          },
+          {
+            text: t('requestForm.heading'),
+            url: returnUrl
+          },
+          { text: t('requestFeedback.viewFeedback') }
+        ]}
+      />
+
+      <PageHeading className="margin-top-6- margin-bottom-1">
+        {t('requestFeedback.heading')}
+      </PageHeading>
+      {returnToFormLink}
+
+      {sortBy(
+        request.feedback.filter(
+          e => e.action === TRBFeedbackAction.REQUEST_EDITS
+        ),
+        'createdAt'
+      )
+        .reverse()
```

Comment:
This is just a nitpick, but you could use orderBy() here and specify the sort order instead of using sortBy().reverse().

```
orderBy(
    request.feedback.filter(
        e => e.action === TRBFeedbackAction.REQUEST_EDITS
    ),
    'createdAt',
    'desc'
)
```

Reply:
It's the same thing. I used sortBy since orderBy can also sort on multiple props and this doesn't need that.

Reply:
Ok. I just want to leave it.

I found out after committing that, that backend actually already seems to return it sorted, but it's just not descending. I was eventually going to bring this up, and if backend can do that, then I can update the query instead of doing any list handling.